pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -125,6 +141,27 @@ func TestPipeline_Validate_Failure(t *testing.T) {
 			Message: `expected at least one, got none`,
 			Paths:   []string{"spec.description", "spec.params", "spec.resources", "spec.tasks", "spec.workspaces"},
 		},
+	}, {
+		name: "pipelinetask custom task references missing apiversion",
+		p: &Pipeline{
+			ObjectMeta: metav1.ObjectMeta{Name: "pipeline"},
+			Spec: PipelineSpec{
+				Tasks: []PipelineTask{{Name: "foo", TaskRef: &TaskRef{Kind: "Example", Name: ""}}},
```

Comment:
what happens if there was apiversion but no kind?

Reply:
It's tested in another test case, I will move these cases there.